src/List.tsx

```diff
@@ -0,0 +1,34 @@
+function List() {
```

Comment:
Could you please move the LIst component to the components folder? This will help keep our project structure organized and consistent.